question28/js/password.js

```diff
@@ -40,8 +42,21 @@ passwordToggleButtons.forEach((passwordToggleButton) => {
   passwordToggleButton.addEventListener('click', togglePassword); 
 });
 
+const setNewPassword = () => {
+  const newPasswordValue = passwordInputArea.value;
```

Comment:
IMO
`newPasswordValue`
I thought it would be better to pass it on as an argument.
This is to generalize the function.

Reply:
Thank you for your review! 
Fixed to get the value in init() and pass it as an argument to setNewPassword(). Please confirm that my understanding is correct.

Fixed commit : https://github.com/csakyo/homework/pull/45/commits/1e2c6091d81baefabf6d603d2ae5f3769e3b9876